src/memote/utils.jl

```diff
@@ -106,4 +107,19 @@ to_element(x::String) = begin
     elements[sym]
 end
 
+"""
+$(TYPEDSIGNATURES)
+
+A helper function that returns the median upper and lower bounds in a tuple. If none can be calculated
+(-1000.0, 1000.0) are set as the default values.
```

Comment:
```suggestion
A helper function that returns the median upper and lower bounds in a tuple. If none can be calculated,
constants from COBREXA are used as the default values.
```

Avoid magic constants: Take the default value from COBREXA constants, or have the -1000/1000 values in the config.

Reply:
I just changed it to use the bound_default from reaction_config, I hope that's alright since I couldn't find the necessary constants in base/constants.jl within COBREXA unfortunately

Reply:
I think it is this one https://github.com/LCSB-BioCore/COBREXA.jl/blob/master/src/base/constants.jl#L8
should be accessible as `COBREXA._constants.default-reaction_bound`.

Reply:
Oops sorry I totally missed that :(
l'll change it now